src/components/organisms/about-content/about-content.tsx

```diff
@@ -1,38 +1,121 @@
 import Card from "components/atoms/card/card";
 import DefaultText from "components/atoms/typography/default-text";
 import HeaderText from "components/atoms/typography/heading-text";
+import DefaultButton from "components/atoms/button/button";
+import {
+  Stack,
+  Flex,
+  Button,
+  Text,
+  VStack,
+  Divider,
+  Box
+  
+} from "@chakra-ui/react";
+import Link from "components/atoms/link/link";
+import Icon from "components/atoms/icon/icon";
+import { 
+  SiTwitter,
+  SiLinkedin,
+  SiFacebook
+} from "react-icons/si";
+
 
 const AboutContent = () => {
   return (
-    <Card section className="flex min-h-full flex-col gap-8">
-      <>
-        <article className="flex flex-col gap-2">
-          <HeaderText level="h2">What is TechIsHiring?</HeaderText>
-          <DefaultText as="p">
-            TechIsHiring is an organization who&apos;s mission is to help great
-            Engineers and great jobs find each other. We try to accomplish this
-            by focusing less on advertising job opportunities but by encouraging
-            meaningful interactions between someone searching for job
-            opportunities and those who can help.
-          </DefaultText>
-        </article>
-        <article className="flex flex-col gap-2">
-          <HeaderText level="h2">TechIsHiring&apos;s Story</HeaderText>
-          <DefaultText as="p">
-            TechIsHiring was born on Twitter when{" "}
-            <a href="www.twitter.com/Chad_R_Stewart" target="_blank">
-              Chad Stewart
-            </a>
-            , the Founder of TechIsHiring noticed during the pandemic that a lot
-            of people were tweeting searching for job opportunities and posting
-            opportunities that they had came across. There was no centralized
-            place for these tweets and so &apos;#TechIsHiring&apos; and the{" "}
-            <a href="www.twitter.com/TechIsHiring">TechIsHiring account</a> was
-            born.
-          </DefaultText>
-        </article>
-      </>
-    </Card>
+    <Box w={"full"}>
+      <Flex
+        w={"full"}
+        h={{base: "100vh", md:"80vh"}}
+        backgroundImage={ 
+          { base: "url(https://res.cloudinary.com/phantom1245/image/upload/v1676232320/techishiring/Rectangle_1336_ncjum3.png)",
+          md: "url(https://res.cloudinary.com/phantom1245/image/upload/v1676232321/techishiring/Rectangle_1335_dzx45d.png)" }
+        }
```

Comment:
Please do not hard-code urls of images. Use the image local to the project when possible.

Reply:
okay noted